image-layout.md

```diff
@@ -2,6 +2,7 @@
 
 * The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 * This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
+* This section defines the `application/vnd.oci.image.layout.v1+json` [media type](media-types.md).
```

Comment:
> I think this line should be moved down here to the oci-layout-file section. Otherwise it seems to imply that the entire document is talking about this media type.

I'm a little doubting on it. It seems had better to be aligned with other spec docs (manifest, index, config..), in which `define ... media type` is put on the first section.